### PR TITLE
Add support for exporting hand-written simple models to backend test suite

### PR DESCRIPTION
**Pull request by bddppq**
@guoyuhong

### CODE REVIEW

File: onnx/backend/test/loader/__init__.py [hunk 1]
```diff
@@ -52,11 +52,17 @@ def load_node_tests(data_dir=os.path.join(DATA_DIR, 'node')):
     return testcases
 
 
-def _load_model_tests(data_dir):
+
+def load_model_tests(data_dir=os.path.join(DATA_DIR, 'model'), kind=None):
```
Comment (houseroad) [inline onnx/backend/test/loader/__init__.py]: We have load_model_tests, load_node_tests, collect_tests (in both case.model and case.node). Some comments can help us remember the purpose of the functions.

File: onnx/backend/test/cmd_tools.py [hunk 2]
```diff
@@ -42,14 +42,34 @@ def prepare_dir(path):
...
+        if case.kind == 'real':
+            with open(os.path.join(output_dir, 'data.json'), 'w') as f:
+                json.dump({
+                    'url': case.url,
+                    'model_name': case.model_name,
+                }, f, sort_keys=True)
+        else:
+            with open(os.path.join(output_dir, 'model.pb'), 'wb') as f:
+                f.write(case.model.SerializeToString())
+            for i, (inputs, outputs) in enumerate(case.data_sets):
```
[the remark this answers is not in the document]
Comment (bddppq) [reply onnx/backend/test/cmd_tools.py]: @houseroad Yep but need to first makes node testcases accept multiple sets of inputs&outputs.

File: onnx/backend/test/loader/__init__.py (hunk 1 above)
Comment (bddppq) [reply onnx/backend/test/loader/__init__.py]: Added

**Comment by guoyuhong** [comment]
@bddppq After this change, I found "backend-test-tools generate-data" will not cause difference for the node test. So I don't need to add the data source now?

**Comment by bddppq** [comment]
@guoyuhong You do need. Maybe your working copy is at the wrong branch/version?

**Pull request merged**